lib/node-loader-babel.js

```diff
@@ -43,9 +51,9 @@ export async function load(url, context, defaultLoad) {
 
     return {
       source: transformed.code,
-      // Maybe a shaky assumption
-      // TODO: look at babel config to see whether it will output ESM/CJS or other formats
-      format: "module",
+      // NOTE: transform-modules-commonjs doesn't work properly.
```

Comment:
Which bug in transform-modules-commonjs you're referring to?

I don't think we ever need to forcibly change the module format - see my comment above about how I don't think babel will change module format unless explicitly told to do so via plugin, which is something controlled via babel config.